sweetalert2.d.ts

```diff
@@ -322,6 +322,9 @@ declare module 'sweetalert2' {
          * @default 'center'
          */
         position?:
+            'top' | 'top-start' | 'top-end' |
+            'center' | 'center-start' | 'center-end' |
+            'bottom' | 'bottom-start' | 'bottom-end' |
             'top' | 'top-left' | 'top-right' |
             'center' | 'center-left' | 'center-right' |
             'bottom' | 'bottom-left' | 'bottom-right';
```

Comment:
There are duplicates in this list of types. Is that desired (simmetry I assume, but catched my eye)?

Reply:
Oops, removed, thanks for catching this.